src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToReferenceAnAssembly.cs

```diff
@@ -90,6 +90,198 @@ public static void Main()
                 .And.HaveStdOutContaining("Hello from a direct reference.");
         }
 
+        [Theory]
+        [InlineData("netcoreapp2.0", "netstandard2.0", "net40")]
+        [InlineData("netcoreapp2.0", "netstandard2.0", "netstandard1.5")]
+        [InlineData("netcoreapp2.0", "netstandard2.0", "netcoreapp1.0")]
+        public void ItRunsAppsReferencingAProjectDirectlyReferencingAssemblies(
```

Comment:
Looks like both tests have a ProjectReference between app and dll references. I think we should have a test with no ProjectReferences but with indirect dll deployed via Reference. I think that because discussing that change led to fixes in the product code. Somebody might reintroduce the same wrong assumption without test coverage about what metadata to expect.

Reply:
Fixed.